jsons/Units.json

```diff
@@ -41,11 +41,25 @@
 		"movement": 1,
 		"cost": 35,
 		"requiredTech": "Bronze Working",
-		"requiredResource": "Copper", // TODO: Requires Copper OR Iron
 		"uniques": [
 			"Can upgrade to [Maceman]",
-			"Consumes [1] [Iron]",
-			"[+50]% Strength <vs [Melee] units>"
+			"[+50]% Strength <vs [Melee] units>",
+
+			// Requires Copper OR Iron
+			"Unavailable <when number of [Copper] is less than [1]> <when number of [Iron] is less than [1]> <hidden from users>",
+			"Consumes [1] [Copper] <when number of [Copper] is more than [0]> <when number of [Iron] is less than [1]> <hidden from users>",
+			"Consumes [1] [Iron] <when number of [Copper] is less than [1]> <when number of [Iron] is more than [0]> <hidden from users>",
+			"Consumes [1] [Copper] <when number of [Copper] is more than [0]> <when number of [Iron] is more than [0]> <hidden from users>"
```

Comment:
Surely you could remove the second conditional both here and above, and have it be 2 uniques instead of 3?

But nice workaround!! Very clever